src/Router.ts

```diff
@@ -71,7 +88,7 @@ export class RouterLink extends VNode {
 
     clickFunction(event: Event, link: VNode) {
         const ln = link as RouterLink;
-        if (ln.app.router.componentMap.has(ln.target)) {
+        if (ln.app.router.hasRouteRegistered(ln.target)) {
             history.pushState({}, "", document.location.pathname)
             ln.app.router.resolveRoute(ln.target);
```

Comment:
Catch here and log to console.error to supress Browser generated warning about uncaught exception